src/arload.py

```diff
@@ -43,6 +43,8 @@ def argflag_init():
                 'extract':dict({'binby':1.0}),
                 'load':dict({'extracted':False, 'calibrated':False}),
                 'calibrate':dict({'cwpolyorder':2, 'threshold':3.0, 'polyorderpri':4, 'polyordersec':8, 'pcapri':[4,2,2,0], 'pcasec':[5,4,3,2,1,1], 'detection':6.0, 'method':'simple', 'nfitpix':7, 'idfile':'wave_ThAr_3100-11000.npy', 'linelist':'arclist.ThAr', 'numsearch':20, 'sigmacut':2.0}),
```

Comment:
I would have expected `id_pix` and `id_wave` to go in the calibrate dict, above. Then, the input file will need the following lines to set the pixels and wavelengths.

arc calibrate id_pix 220.63808792,417.17122925,562.95948074,669.07189712,884.38191292
arc calibrate id_wave 5868.4165,6334.4276,6678.2766,6929.4672,7438.8981